modules/twinklearv.js

```diff
@@ -18,14 +18,57 @@ Twinkle.arv = function twinklearv() {
 		return;
 	}
 
-	var title = mw.util.isIPAddress(username) ? 'Report IP to administrators' : 'Report user to administrators';
+	var isIP = mw.util.isIPAddress(username);
+	var title = isIP ? 'Report IP to administrators' : 'Report user to administrators';
 
 	Twinkle.addPortletLink(function() {
-		Twinkle.arv.callback(username);
+		Twinkle.arv.callback(username, isIP);
 	}, 'ARV', 'tw-arv', title);
 };
 
-Twinkle.arv.callback = function (uid) {
+Twinkle.arv.callback = function (username, isIP) {
+	var params = {
+		format: 'json',
+		action: 'query',
+		list: 'blocks',
+		bkprop: 'range'
```

Comment:
Probably only needs to be added if `isIP`, right?

Reply:
from what I recall, this is ignored if its not an IP